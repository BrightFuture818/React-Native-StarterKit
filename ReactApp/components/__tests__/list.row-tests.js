/**
 * Test to check if the component renderes correctly
 */
'use strict'

import 'react-native'
import React from 'react'
import ListRow from '../list.row'
import renderer from 'react-test-renderer'

/**
 * Check if alerts renders correcly
 * and asserting it to the matching snapshot
 */
it('ListRow renders correcly', () => {

  const tree = renderer.create(
    <ListRow onPress={ e => e}/>
  ).toJSON()

  expect(tree).toMatchSnapshot()

});
