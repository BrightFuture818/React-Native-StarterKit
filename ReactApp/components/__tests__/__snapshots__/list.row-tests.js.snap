exports[`test ListRow renders correcly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#FFF",
      "left": 0,
      "opacity": 1,
      "right": 0,
    }
  }
  testID={undefined}>
  <View
    style={
      Object {
        "borderBottomColor": "#E7E7E7",
        "borderBottomWidth": 1,
        "paddingVertical": 20,
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#555",
            "fontFamily": "Avenir",
            "fontSize": 14,
            "fontWeight": "500",
            "lineHeight": 21,
          },
          Object {
            "backgroundColor": "transparent",
            "color": "#555",
            "fontWeight": "500",
            "textAlign": "center",
          },
        ]
      }>
      LOREM IPSUM
    </Text>
  </View>
</View>
`;
